src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <section className="containerCalculator">
+        <h3 className="result">0</h3>
+        <div className="buttonsContainer">
+          <button type="button" className="buttonCalculator">AC</button>
+          <button type="button" className="buttonCalculator">+/-</button>
+          <button type="button" className="buttonCalculator">%</button>
+          <button type="button" className="buttonCalculator color-right">÷</button>
+          <button type="button" className="buttonCalculator">7</button>
+          <button type="button" className="buttonCalculator">8</button>
+          <button type="button" className="buttonCalculator">9</button>
+          <button type="button" className="buttonCalculator color-right">x</button>
+          <button type="button" className="buttonCalculator">4</button>
+          <button type="button" className="buttonCalculator">5</button>
+          <button type="button" className="buttonCalculator">6</button>
+          <button type="button" className="buttonCalculator color-right">-</button>
+          <button type="button" className="buttonCalculator">1</button>
+          <button type="button" className="buttonCalculator">2</button>
+          <button type="button" className="buttonCalculator">3</button>
+          <button type="button" className="buttonCalculator color-right">+</button>
+          <button type="button" className="buttonCalculator zero">0</button>
+          <button type="button" className="buttonCalculator">.</button>
+          <button type="button" className="buttonCalculator color-right">=</button>
+        </div>
+      </section>
+    );
+  }
```

Comment:
- [ ] You did a good job on the styles but you need to fix some issues. Please remove the white space between the output and input buttons, and also reduce the font size for the buttons.

### Your design

![image](https://user-images.githubusercontent.com/83803414/213193158-05846422-d8ba-4a94-a2a4-aaa42c681353.png)

### Given design

![image](https://user-images.githubusercontent.com/83803414/213193212-0dec3885-bffb-4b75-a1bb-9698f1a0a695.png)